flink-connector-aws/flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/source/KinesisStreamsSource.java

```diff
@@ -96,11 +96,19 @@ public class KinesisStreamsSource<T>
             KinesisDeserializationSchema<T> deserializationSchema,
             KinesisShardAssigner kinesisShardAssigner,
             boolean preserveShardOrder) {
-        Preconditions.checkNotNull(streamArn);
+        Preconditions.checkNotNull(
+                streamArn,
+                "No stream ARN was supplied to the KinesisStreamsSource builder.");
         Preconditions.checkArgument(!streamArn.isEmpty(), "stream ARN cannot be empty string");
-        Preconditions.checkNotNull(sourceConfig);
-        Preconditions.checkNotNull(deserializationSchema);
-        Preconditions.checkNotNull(kinesisShardAssigner);
+        Preconditions.checkNotNull(
+                sourceConfig,
+                "No source config was supplied to the KinesisStreamsSource builder.");
+        Preconditions.checkNotNull(
+                deserializationSchema,
+                "No KinesisDeserializationSchema was supplied to the KinesisStreamsSource builder.");
+        Preconditions.checkNotNull(
+                kinesisShardAssigner,
+                "No KinesisShardAssigner was supplied to the  KinesisStreamsSource builder.");
```

Comment:
nit:
```suggestion
                "No KinesisShardAssigner was supplied to the KinesisStreamsSource.");
```
or
```suggestion
                "Can't create KinesisStreamsSource with null KinesisShardAssigner");
```

same for all

Reply:
Thanks, I choose the first option since this is used in another place as well.

I tried to fix the spotless errors, but didn't figure out how to do spotless:apply or how to run it locally.